Vikta-UI-testing-sample/src/test/java/com/griddynamics/qa/vikta/uitesting/sample/stepsDefinitions/BaseSteps.java

```diff
@@ -50,6 +51,13 @@ TestDataAndProperties getData() {
     return PageFactory.initElements(getDriver(), pageClass);
   }
 
+  @Step
+  public void scroll(WebElement lastImage) {
```

Comment:
you can call the parameter something like 'targetElement'. Since it's  in BaseSteps it can be used in other cases than last image